app/models/dataset/Dataset.scala

```diff
@@ -664,28 +664,26 @@ class DatasetDAO @Inject()(sqlClient: SqlClient, datasetLayerDAO: DatasetLayerDA
   }
 }
 
-class DatasetResolutionsDAO @Inject()(sqlClient: SqlClient)(implicit ec: ExecutionContext)
-    extends SimpleSQLDAO(sqlClient) {
-  private def parseRow(row: DatasetResolutionsRow): Fox[Vec3Int] =
+class DatasetMagsDAO @Inject()(sqlClient: SqlClient)(implicit ec: ExecutionContext) extends SimpleSQLDAO(sqlClient) {
+  private def parseRow(row: DatasetMagsRow): Fox[Vec3Int] =
     for {
-      resolution <- Vec3Int.fromList(parseArrayLiteral(row.resolution).map(_.toInt)) ?~> "could not parse resolution"
-    } yield resolution
+      mag <- Vec3Int.fromList(parseArrayLiteral(row.mag).map(_.toInt)) ?~> "could not parse mag"
+    } yield mag
 
-  def findDataResolutionForLayer(datasetId: ObjectId, dataLayerName: String): Fox[List[Vec3Int]] =
+  def findMagForLayer(datasetId: ObjectId, dataLayerName: String): Fox[List[Vec3Int]] =
     for {
-      rows <- run(
-        DatasetResolutions.filter(r => r._Dataset === datasetId.id && r.datalayername === dataLayerName).result)
+      rows <- run(DatasetMags.filter(r => r._Dataset === datasetId.id && r.datalayername === dataLayerName).result)
         .map(_.toList)
-      rowsParsed <- Fox.combined(rows.map(parseRow)) ?~> "could not parse resolution row"
+      rowsParsed <- Fox.combined(rows.map(parseRow)) ?~> "could not parse mag row"
     } yield rowsParsed
 
-  def updateResolutions(datasetId: ObjectId, dataLayersOpt: Option[List[DataLayer]]): Fox[Unit] = {
-    val clearQuery = q"DELETE FROM webknossos.dataset_resolutions WHERE _dataset = $datasetId".asUpdate
+  def updateMags(datasetId: ObjectId, dataLayersOpt: Option[List[DataLayer]]): Fox[Unit] = {
+    val clearQuery = q"DELETE FROM webknossos.dataset_mags WHERE _dataset = $datasetId".asUpdate
     val insertQueries = dataLayersOpt.getOrElse(List.empty).flatMap { layer: DataLayer =>
-      layer.resolutions.map { resolution: Vec3Int =>
+      layer.resolutions.map { mag: Vec3Int =>
```

Comment:
The HTTP API is only part of the problem here. The bigger deal is the datasource-properties.json files on disk (and the code that creates and reads them, scala, python and matlab). #7293 describes an idea of how to get around this. However, I did not want to include that in this PR since it’s a more complicated change. It’s also made even more complicated because the MagLocators in e.g. ZarrDataLayers use the keyword "mags", so at least there it is already taken (that probably was a mistake…). I don’t know how to solve this elegantly. I hope we can get rid of datasource-properties.jsons eventually, but I don’t really have a plan for that either.

Reply:
> The bigger deal is the datasource-properties.json files on disk

Ah damn it. Sorry, didn't see this 

> However, I did not want to include that in this PR since it’s a more complicated change.

Sure, makes a lot of sense.